### DISCUSSION TITLE
✅ 🔗 🚀 Feature Request: Easy Exit on new Editor Top Bar (Duplicate of #22359) - [ED-10275]

### DISCUSSION THREAD

**Discussion by oscarhenao** [elementor/elementor#22464] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Not very nice to have to open a new tab on the browser to be able to exit from the Elementor editor to the WP dashboard.

### Describe the solution you'd like

It would be nice to have on the Editor Top Bar feature, easy access to Exit from the Elementor editor to the WP dashboard without opening a new tab as happens by clicking on the Manage Website option.

### Describe alternatives you've considered

ESC is not always working.

**Comment by nicholaszein** [comment]
Hello there, @oscarhenao!

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #22359 

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards,

Nick

**Comment by nicholaszein** [comment]
Hi!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor  v3.14.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
